server/service/getCars.js

```diff
@@ -0,0 +1,48 @@
+//Node JS version of older APIs
+const mysql = require("mysql2");
+const config = require("./config")
+
+const db = mysql.createPool(config.db)
+
+const Service = {
+    getmade: (app)=> {
+        app.get('/made', (req, res) => {
+            const made = "SELECT * FROM carbrands";
+            db.query(made, (err, data) => {
+                if (err) return res.json(err)
```

Comment:
and use like that (err: Interface, data: CarBrands), CarBrands - is your interface to undestand what comes from DB. Apply everywhere this approach